README.md

```diff
@@ -5,6 +5,7 @@
 * [Installing the Client](#installing-the-client)
 * [Using the Client](#using-the-client)
 * [Serialization Considerations](#serialization-considerations)
+* [Code Samples](#code-samples)
```

Comment:
Can we move Code Samples above `Serialization Considerations`? I think `Code Samples` is higher priority than it.